tests/integrations/utilities/rag/data/langchain_readme.md

```diff
@@ -0,0 +1,296 @@
+---
```

Comment:
Is it just copy of https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/langchain_handler/README.md
If so we could use link to it